cmd/removeProcessor.go

```diff
@@ -37,7 +37,8 @@ func newRemoveTransferProcessor(cca *cookedCopyCmdArgs, numOfTransfersPerPart in
 		SourceSAS:      cca.sourceSAS,
 
 		// flags
-		LogLevel: cca.logVerbosity,
+		LogLevel:       cca.logVerbosity,
+		BlobAttributes: common.BlobTransferAttributes{DeleteSnapshotsOption: cca.deleteSnapshotsOption},
```

Comment:
should consolidate the remove processor to use the standard template later

Reply:
Soon :tm:

---
In reply to: [334834728](https://github.com/Azure/azure-storage-azcopy/pull/690#discussion_r334834728) [](ancestors = 334834728)